core/src/style/textStyleBuilder.cpp

```diff
@@ -27,7 +27,9 @@ const static std::string key_name("name");
 
 TextStyleBuilder::TextStyleBuilder(const TextStyle& _style)
     : StyleBuilder(_style),
-      m_style(_style) {}
+      m_style(_style) {
+    m_textLabels = std::make_unique<TextLabels>(m_style);
```

Comment:
with this initialization of `m_textLabels` either is redundant or should be done a `reset` with a newer `TextLabels` instance.